index.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+/**
+ * Allows users to `require('@newrelic/koa')` directly in their app. If they
+ * for some reason choose to explicitly use an older version of our instrumentation
+ * then the supportability metrics for custom instrumentation will trigger.
+ */
+module.exports = function initialize() {
+  var newrelic = require('newrelic')
+  newrelic.instrumentWebframework('koa', require('./lib/instrumentation'))
+}
```

Comment:
Don't export the initialize method, just have the module-level logic perform the hooking. Customers should be able to put _just_ `require('@newrelic/koa')` in their app and have it hook.